### PR TITLE
tests_l2: Update qat workload commands in readme

### PR DESCRIPTION
**Pull request by vbedida79**
Update readme with qatlib workload commands based on https://github.com/intel/intel-technology-enabling-for-openshift/pull/137 and https://github.com/intel/intel-technology-enabling-for-openshift/pull/132

### CODE REVIEW

File: tests/l2/README.md [hunk 1]
```diff
@@ -73,8 +73,10 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
 
 *	Deploy and execute the workload.
 
-    * Run with IPC_LOCK capability for pod
+    * Run with IPC_LOCK capability for pod using custom SCC and rbac with service account
```
Comment (uMartinXu) [inline tests/l2/README.md:76]: Create SCC intel-qat-scc for Intel QAT technology based workload if this SCC is not created

File: tests/l2/README.md [hunk 2]
```diff
@@ -73,8 +73,10 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
...
   
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_scc.yaml```
+
 ```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_rbac.yaml```
```
Comment (uMartinXu) [inline tests/l2/README.md:80]: Create the intel-qat service account using intel-qat-scc

File: tests/l2/README.md [hunk 3]
```diff
@@ -73,8 +73,10 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
...
 
 ```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/qat/qatlib_job.yaml```
```
Comment (uMartinXu) [inline tests/l2/README.md:82]: deploy the qatlib workload job with intel-qat service account

File: tests/l2/README.md (hunk 2 above)
Comment (vbedida79) [reply tests/l2/README.md]: changed it to Create the intel-qat service account to use intel-qat-scc

File: tests/l2/README.md (hunk 1 above)
Comment (vbedida79) [reply tests/l2/README.md]: Create SCC intel-qat-scc for Intel QAT based workload, if this SCC is not created

File: tests/l2/README.md [hunk 4]
```diff
@@ -73,11 +73,19 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
 
 *	Deploy and execute the workload.
 
-    * Run with IPC_LOCK capability for pod
-  
-```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_rbac.yaml```
+    * Create SCC intel-qat-scc for Intel QAT based workload, if this SCC is not created
+        
+      ```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_scc.yaml```
```
Comment (uMartinXu) [inline tests/l2/README.md:78]: remove the spaces in the beginning of the line.

File: tests/l2/README.md [hunk 5]
```diff
@@ -73,11 +73,19 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
...
+      
+
+    * Create the intel-qat service account to use intel-qat-scc
+
+      ```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/security/qatlib_rbac.yaml```
+
+
```
Comment (uMartinXu) [inline tests/l2/README.md:85]: remove this empty line

File: tests/l2/README.md [hunk 6]
```diff
@@ -73,11 +73,19 @@ This workload runs [qatlib](https://github.com/intel/qatlib) sample tests. Refer
...
+    * Deploy the qatlib workload job with intel-qat service account
 
-```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/qat/qatlib_job.yaml```
+      ```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/tests/l2/qat/qatlib_job.yaml```
```
Comment (uMartinXu) [inline tests/l2/README.md:88]: remove the spaces in the beginning of the line

**Comment by vbedida79** [comment]
updated all changes, please check

**Comment by uMartinXu** [comment]
Looks good to me. Thanks!

**Pull request merged**